bucket/harmonoid.json

```diff
@@ -0,0 +1,96 @@
+{
+    "version": "0.3.6",
+    "homepage": "https://harmonoid.com/",
+    "description": "Plays & manages your music library. Looks beautiful & juicy. Playlists, visuals, synced lyrics, pitch shift, volume boost & more.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://raw.githubusercontent.com/harmonoid/harmonoid/master/EULA.txt"
+    },
+    "notes": [
+        "To install harmonoid file associations, run:",
+        "install-harmonoid-associations.ps1",
+        "To install the harmonoid context menu, run:",
+        "install-harmonoid-context.ps1",
+        "",
+        "To uninstall harmonoid file associations, run:",
+        "uninstall-harmonoid-associations.ps1",
+        "To uninstall the harmonoid context menu, run:",
+        "uninstall-harmonoid-context.ps1"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/alexmercerind2/harmonoid-releases/releases/download/v0.3.6/harmonoid-windows-exe.zip",
+            "hash": "b2e169b1fc337e2214c9181db2f7035d8555af347acd9fbbbb96a4bbf564da1b"
+        }
+    },
+    "extract_dir": "harmonoid-windows-exe",
+    "pre_install": [
+        "$newDir = $dir.Replace($version, 'current')",
+        "",
+        "$scriptsDir = \"$bucketsdir\\extras\\scripts\"",
+        "",
+        "Get-ChildItem \"$scriptsDir\\harmonoid\\*.ps1\" | ForEach-Object {",
+        "    $content = ($_ | Get-Content).Replace('CURRENT_DIRECTORY', $newDir)",
+        "    Set-Content \"$dir\\$($_.Name)\" $content -Encoding 'utf8' -Force",
+        "}"
+    ],
+    "bin": [
+        "harmonoid.exe",
+        "install-harmonoid-associations.ps1",
+        "uninstall-harmonoid-associations.ps1",
+        "install-harmonoid-context.ps1",
+        "uninstall-harmonoid-context.ps1"
+    ],
```

Comment:
```suggestion
    "bin": "harmonoid.exe",
```

Reply:
Why would I remove that from the bin? It doesn't make sense why would I want to take away the convenience away from the user?

Reply:
Installing and removing associations is a one-time procedure - no need to pollute the shims directory for that. The `notes` section has also been updated above to use the full path of the script.